website/docs/r/docdb_cluster.html.markdown

```diff
@@ -0,0 +1,115 @@
+---
+layout: "aws"
+page_title: "AWS: aws_docdb"
+sidebar_current: "docs-aws-resource-docdb-cluster"
+description: |-
+  Manages a DocDB Aurora Cluster
+---
+
+# aws_docdb_cluster
+
+Manages a DocDB Cluster.
+
+Changes to a DocDB Cluster can occur when you manually change a
+parameter, such as `port`, and are reflected in the next maintenance
+window. Because of this, Terraform may report a difference in its planning
+phase because a modification has not yet taken place. You can use the
+`apply_immediately` flag to instruct the service to apply the change immediately
+(see documentation below).
+
+~> **Note:** using `apply_immediately` can result in a brief downtime as the server reboots.
+~> **Note:** All arguments including the username and password will be stored in the raw state as plain-text.
+[Read more about sensitive data in state](/docs/state/sensitive-data.html).
+
+## Example Usage
+
+```hcl
+resource "aws_docdb_cluster" "docdb" {
```

Comment:
This example should include `skip_final_snapshot` or `final_snapshot_identifier` so it is a valid configuration. 👍